lib/message/twitch-types/usernotice.ts

```diff
@@ -224,7 +224,16 @@ export class UsernoticeMessage extends ChannelIRCMessage {
   public readonly emotes: TwitchEmoteList;
   public readonly emotesRaw: string;
 
+  /**
+   * Can be an array of Twitch AutoMod flagged words, for use in moderation and/or filtering purposes.
+   *
+   * **Note:** This is an undocumented Twitch IRC feature and may change at any time, use at your own risk.
+   */
   public readonly flags: TwitchFlagList | undefined;
+
+  /**
+   * Twitch AutoMod raw flags string.
```

Comment:
```suggestion
   * Twitch AutoMod raw flags string.
   *
   * If the `flags` tag is missing or of a unparseable format, this will be `undefined`. This is unlike most other
   * attributes which when missing or malformed will fail the message parsing. However since this attribute is
   * completely undocumented we cannot rely on the `flags` tag being stable, so this soft fallback is used instead.
   * In short, ensure your implementation can handle the case where this is `undefined` or is in
   * a format you don't expect.
```